### DISCUSSION TITLE
Error: Could not load /id-__x00__react-jsx-dev-runtime-NY5FA.js

### DISCUSSION THREAD

**Discussion by SanujBansal** [crxjs/chrome-extension-tools#461] [General]
Hey everyone!
Does some body get this following error. I just followed up the basic instructions from here:- https://crxjs.dev/vite-plugin/getting-started/react/create-project
![image](https://user-images.githubusercontent.com/32351832/183026837-dcd0b959-ac6a-438c-afc2-5584bb92930a.png)

**Comment by jacksteamdev** [comment]
This is a known issue with Vite 3, see #454

If you install Vite 2 and compatible plugins, things should work as expected:

```sh
npm i -D vite@^2 @vitejs/plugin-react@^1
```
